src/main/webapp/confirmation.js

```diff
@@ -20,7 +20,7 @@ function getScheduledSessions() {
 //Helper function for getScheduledSessions, used for testing
 async function getScheduledSessionsHelper(window) {
     await fetch('/confirmation', {method: 'GET'}).then((response) => {
-        //if the student id is not the id of the current user
+        //if the student is not the current user/signed in
```

Comment:
I understand what you're trying to say here but I think it'd make more sense to say something like "if the student is not the current user or is not signed in"